pkg/upload/all.go

```diff
@@ -20,18 +20,19 @@ func All(
 	overwrite bool,
 	apiKey string,
 	dryRun bool,
+	logger log.Logger,
 ) error {
 
 	// Build the file list from the path(s)
-	log.Info("building file list...")
+	logger.Info("building file list...")
 
 	fileList, err := utils.BuildFileList(paths)
 
 	if err != nil {
-		log.Error(" error building file list", 1)
+		logger.Fatal(" error building file list")
 	}
 
-	log.Info("File list built..")
+	logger.Info("File list built..")
```

Comment:
Feels like the two `Info` log calls are unnecessary, when comparing this to other commands. Just an error message if it fails seems sufficient.

Likewise with the Dart command.